Tasks/RunJMeterLoadTestV1/Start-ApacheJMeterTest.ps1

```diff
@@ -184,5 +184,12 @@ else
 	("Connection '{0}' failed for service '{1}'" -f $connectedServiceName, $connectedServiceDetails.Url.AbsoluteUri) >> $summaryFile
 }
 
+$featureFlags = @{
+	failDeprecatedBuildTask  = [System.Convert]::ToBoolean($env:FAIL_DEPRECATED_BUILD_TASK)
+}
+if ($featureFlags.failDeprecatedBuildTask)
+{
+	throw "The ApacheJMeterLoadTest@1 (Cloud-based Apache JMeter load test) task haas been deprecated since June 4, 2019 and will soon be retired. For cloud-based load testing, use the AzureLoadTest@1 task."
```

Comment:
Shouldn't this string be sourced from task.json?

Reply:
Same as above